### PR TITLE
release(v1.3.0-beta.2): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.3.0-beta.2 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: website/content/v1.3/reference/cli.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ talosctl cluster create [flags]
       --bad-rtc                                  launch VM with bad RTC state (QEMU only)
       --cidr string                              CIDR of the cluster network (IPv4, ULA network for IPv6 is derived in automated way) (default "10.5.0.0/24")
       --cni-bin-path strings                     search path for CNI binaries (VM only) (default [/home/user/.talos/cni/bin])
-      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.3.0-beta.1/talosctl-cni-bundle-${ARCH}.tar.gz")
+      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.3.0-beta.2-dirty/talosctl-cni-bundle-${ARCH}.tar.gz")
```
Comment (frezbo) [inline website/content/v1.3/reference/cli.md:99]: -dirty

**Review by frezbo — APPROVED** [review]

File: website/content/v1.3/reference/cli.md (hunk 1 above)
Comment (smira) [reply website/content/v1.3/reference/cli.md:99]: no way :facepalm:

Comment (smira) [reply website/content/v1.3/reference/cli.md]: thank you, I always miss whether it's `make generate docs` or `make docs generate`. Should be the latter.

**Pull request merged**